index.js

```diff
@@ -1,4 +1,3 @@
-/* jslint node: true */
 'use strict';
 
 module.exports = require( './lib/cacheWrapper' );
```

Comment:
We can bin this index file and just set the `main` entry in the `package.json` to point to `lib/cacheWrapper`. See https://github.com/holidayextras/plugin-tapestry/blob/master/package.json#L15 for an example.